src/components/StoreList/StoreList.js

```diff
@@ -1,13 +1,44 @@
 import React from 'react';
 import ButtonList from '../ButtonList/ButtonList';
 import getStore from '../../utils/honestyStore.js';
-import sendSlackMessage from '../../utils/slack';
 
 export default class StoreList extends React.Component {
 	state = {
 		storeList: []
 	};
 
+	usernameToID = username => {
+		console.log(this.props.users);
+		let users = this.props.users;
+		let i = users.length;
+		while (i--) {
+			if (
```

Comment:
We could use `Array.prototype.find` here which could simplify this a bit by not recreating the wheel

```
const user = this.props.users.find(user => user.name === username || user.profile.real_name === username)

return user ? user.id : null
```